
int8 UNKNOWN=-1
int8 EXPLORERMAP=0
int8 SWEEPERMAP=1
int8 UWBMAP=2
int8 SLAMMAP=3
int8 LOCALSLAMMAP=4

int8 kind
